### ISSUE TITLE
Add more stats

### ISSUE THREAD

**Issue by marcoXbresciani** [kbilsted/KeyboordUsage#5]
Unfortunately I'm not a C programmer so I can't be useful...
Anyway having more stats could be very nice: total number of keypresses, average per day/month/week/whatever, speed (could be done?), keys distribution (most used, least used, ...).

Wondering if I could replicate this project with Java... Maybe not.

**Comment by kbilsted** [comment]
If you know java c# will be straight forward. You never get to be too old for a challenge :-)

**Comment by marcoXbresciani** [comment]
Indeed. And I'd love to study it... But my three years old son probably won't allow. No time... No soon, at least.

Ottieni Outlook per Android<https://aka.ms/ghei36>

________________________________

**Comment by kbilsted** [comment]
Not sure exactly what stats you are missing or how you want them presented. There are already some stats

**Comment by marcoXbresciani** [comment]
An interesting set of statistics might be:
- total number of keys pressed;
- average keys pressed per day (or month or week);
- keys combinations distribution (like the current single keys usage distirbution on the left side panel, but for key combinations instead of single keys);
- speed (WPM: words per minute?), if feasible... the WPM definition counts a word each 5 keys (see https://en.wikipedia.org/wiki/Words_per_minute)

**Comment by kbilsted** [comment]
Thanks for the feed back. Im not sure what use those stats provide? Other than being fun to look at. Ie. Not sure they are worth implementing. At least not if i am to implement them;)

**Comment by marcoXbresciani** [comment]
Well, actually they might be useful to track, for example, the most used key combinations in order to, for example, configure or program specific macros, so reducing the use of combinations, that are usually not so good for fingers and wrists.

Ottieni Outlook per Android<https://aka.ms/ghei36>

________________________________

**Comment by kbilsted** [comment]
Good ideas
